From a7ce401ad3039cac1d2a18a24fdbb2f316dd5e4b Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Thu, 13 Jun 2024 10:26:25 +0000
Subject: [PATCH 05/24] rdk/snr: drop NAPI_POLL_WEIGHT

kernel commit b48b89f9c189 ("net: drop the weight argument from netif_napi_add").

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_base.c | 3 +--
 klm/cpk/ice_main.c | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/klm/cpk/ice_base.c b/klm/cpk/ice_base.c
index b1bd0ef..131f93e 100644
--- a/klm/cpk/ice_base.c
+++ b/klm/cpk/ice_base.c
@@ -119,8 +119,7 @@ static int ice_vsi_alloc_q_vector(struct ice_vsi *vsi, u16 v_idx)
 	 * handler here (i.e. resume, reset/rebuild, etc.)
 	 */
 	if (vsi->netdev)
-		netif_napi_add(vsi->netdev, &q_vector->napi, ice_napi_poll,
-			       NAPI_POLL_WEIGHT);
+		netif_napi_add(vsi->netdev, &q_vector->napi, ice_napi_poll);
 
 	/* tie q_vector and VSI together */
 	vsi->q_vectors[v_idx] = q_vector;
diff --git a/klm/cpk/ice_main.c b/klm/cpk/ice_main.c
index d8963cb..fd9c6ef 100644
--- a/klm/cpk/ice_main.c
+++ b/klm/cpk/ice_main.c
@@ -2110,7 +2110,7 @@ static void ice_napi_add(struct ice_vsi *vsi)
 
 	ice_for_each_q_vector(vsi, v_idx)
 		netif_napi_add(vsi->netdev, &vsi->q_vectors[v_idx]->napi,
-			       ice_napi_poll, NAPI_POLL_WEIGHT);
+			       ice_napi_poll);
 }
 
 /**
-- 
2.39.3

